config.yml

```diff
@@ -0,0 +1,246 @@
+      - aws --version
+      - docker --version
+      - docker-compose --version
+      # - --privileged -d --name dind-aws alpine:latest
+      # - docker ps
+      - *ci_helper_test
+
+golang:
+  "1.11":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      GLIDE_VERSION: v0.13.1
+      MODD_VERSION: 0.5
+    cmd_test:
+      - aws --version
+      - go version
+      - glide --version
+      - gin --version
+      - modd --version
+      - *ci_helper_test
+
+java_test: &java_test
+  - java -version
+  - mvn --version
+  - aws --version
+  - modd --version
+  - dot -V
+  - jq --version
+
+java:
+  "6":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      MAVEN_VERSION: 3.2.1
+    cmd_test:
+      - java -version
+      - mvn --version
+    dockerfile_dir: java/6
+  "8":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      MODD_VERSION: 0.5
+    cmd_test: *java_test
+    template_vars:
+      VERSION: 8
+      JAVA_VERSION: jdk8u212-b04-debian-slim
+  "11":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      MODD_VERSION: 0.5
+    cmd_test: *java_test
+    template_vars:
+      VERSION: 11
+      JAVA_VERSION: jdk-11.0.3_7-debian-slim
+
+node_cmd_test: &node_cmd_test
+  - *ci_helper_test
+  - node --version
+  - npm --version
+  - sass --version
+  - modd --version
+
+node:
+  "6.17":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      NODE_VERSION: 6.17.1
+      NPM_VERSION: 3.10.10
+      NVM_VERSION: 0.34.0
+      MODD_VERSION: 0.5
+    cmd_test: *node_cmd_test
+  "8.16":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      NODE_VERSION: 8.16.0
+      NPM_VERSION: 6.4.1
+      NVM_VERSION: 0.34.0
+      MODD_VERSION: 0.5
+    cmd_test: *node_cmd_test
+  "10.15":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      NODE_VERSION: 10.15.3
+      NPM_VERSION: 6.4.1
+      NVM_VERSION: 0.34.0
+      MODD_VERSION: 0.5
+    cmd_test: *node_cmd_test
+
+php_cmd_test: &php_cmd_test
+  - *ci_helper_test
+  - php --version
+  - composer --version
+  - modd --version
+  - security-checker --version
+  - -v $(pwd)/php:/tmp %s php /tmp/test.php
+
+php:
+  "7.1":
+    build_args:
+      APCU_VERSION: 5.1.17
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      COMPOSER_VERSION: 1.8.5
+      MEMCACHED_VERSION: 3.1.3
+      MODD_VERSION: 0.5
+      REDIS_VERSION: 4.3.0
+      SECURITY_CHECKER_VERSION: 5.0.3
+      SSH2_VERSION: 1.1.2
+      XDEBUG_VERSION: 2.7.2
+    cmd_test: *php_cmd_test
+    template_vars:
+      PHP_VERSION: 7.1.29-cli-alpine
+  "7.2":
+    build_args:
+      APCU_VERSION: 5.1.17
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      COMPOSER_VERSION: 1.8.5
+      MEMCACHED_VERSION: 3.1.3
+      MODD_VERSION: 0.5
+      REDIS_VERSION: 4.3.0
+      SECURITY_CHECKER_VERSION: 5.0.3
+      SSH2_VERSION: 1.1.2
+      XDEBUG_VERSION: 2.7.2
+    cmd_test: *php_cmd_test
+    template_vars:
+      PHP_VERSION: 7.2.18-cli-alpine
+
+python:
```

Comment:
Both Python images install CI Helper, so you need to test it.

Just remember that only Java 6 and Terraform come without CI Helper installed.